GitCommands/Git/GitModule.cs

```diff
@@ -595,6 +596,30 @@ public void SaveBlobAs(string saveAs, string blob)
             stream.Write(blobData, 0, blobData.Length);
         }
 
+        public void SaveTreeAs(string saveAs, string hash)
+        {
+            string archivePath = Path.Join(saveAs.SubstringUntilLast('\\'), $"{hash}.zip");
```

Comment:
Use a portable function for the extraction of the path part, please.

Reply:
Thanks, using new method "GetArchivePath" in PathUtils